test/integration/project/tsconfig.base.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "noImplicitAny" : true,
+        "noImplicitThis" : true,
+        "alwaysStrict" : true,
+        "strictNullChecks": true,
```

Comment:
Strict flags (and others) are tsc flags and should not be tested by our compiler.